compiler/src/dotty/tools/dotc/typer/ProtoTypes.scala

```diff
@@ -568,7 +569,7 @@ object ProtoTypes {
       case poly: PolyType =>
         normalize(constrained(poly).resultType, pt)
       case mt: MethodType =>
-        if (mt.isImplicitMethod) normalize(resultTypeApprox(mt), pt)
+        if (mt.isImplicitMethod) normalize(resultTypeApprox(mt, wildcardOnly = true), pt)
```

Comment:
The documentation above still says "if result type depends on implicit parameter, replace with fresh type dependent parameter."